packages/lib-classifier/src/components/Classifier/components/MetaTools/components/Metadata/components/MetadataModal/MetadataModal.js

```diff
@@ -72,6 +76,7 @@ export default function MetadataModal ({
     >
       <Box height='medium' overflow='auto'>
         <StyledDataTable
+          pad='15px'
```

Comment:
Can this use a themed size like `small` or `xsmall`?

Reply:
Unfortunately no because `small` padding is 20px and `xsmall` padding is 10px.